buildConfiguration.xml

```diff
@@ -19,6 +19,8 @@
       <project name="Microsoft.IdentityModel.KeyVaultExtensions" />
       <project name="Microsoft.IdentityModel.ManagedKeyVaultSecurityKey" />
       <project name="Microsoft.IdentityModel.Validators" />
+      <project name="Microsoft.IdentityModel.Abstractions" />
+      <project name="Microsoft.IdentityModel.TestExtensions" />
```

Comment:
Do we want to release this package?

Reply:
<project name="Microsoft.IdentityModel.Abstractions" /> needs to be released. Not sure about TestExtensions. It was copied from Tim's Telemetry PR.

Reply:
Adding TestExtensions to buildConfiguration.xml will result in packing/releasing it. Not sure if that's something that we want to do.

Reply:
I believe it's needed to be released if we want folks to be able to consume it correct? the whole conceit of TestExentions was so folks could more easily reference it to write unit tests. If you think there's a better way to expose that I'm open to that.

Reply:
Do we have docs and samples on how to use TestExtensions?